src/main/scala/org/broadinstitute/dsde/workbench/sam/db/tables/UserTable.scala

```diff
@@ -16,7 +16,7 @@ object UserTable extends SQLSyntaxSupport[UserRecord] {
   def apply(e: ResultName[UserRecord])(rs: WrappedResultSet): UserRecord = UserRecord(
     rs.get(e.id),
     rs.get(e.email),
-    rs.get(e.googleSubjectId),
+    rs.stringOpt(e.googleSubjectId).map(GoogleSubjectId),
```

Comment:
Ran into an issue when using this function to load an entire user in `loadUser` where the GoogleSubjectId was null and `rs.get(e.googleSubjectId)` should've generated `None` but was instead generating `Some(null)`. I found this to be a suitable replacement and I made the same change to other optional values in other tables.